tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -200,21 +112,13 @@ def test_post_sms_notification_uses_inbound_number_as_sender(notify_api, client,
     assert response.status_code == 201
     resp_json = json.loads(response.get_data(as_text=True))
     assert validate(resp_json, post_sms_response) == resp_json
-    notifications = Notification.query.all()
-    assert len(notifications) == 1
-    notification_id = notifications[0].id
-    assert resp_json["id"] == str(notification_id)
-    assert resp_json["content"]["from_number"] == "1"
-    assert notifications[0].reply_to_text == "1"
-    mocked.assert_called_once_with([str(notification_id)], queue="send-sms-tasks")
+    assert mocked.called
 
 
 def test_post_sms_notification_uses_inbound_number_reply_to_as_sender(notify_api, client, notify_db_session, mocker):
```

Comment:
I'm confused as to what the difference is between this test and the previous... but there must be one since the first test previously used `deliver_sms` while this one used `deliver_throttled_sms`.... does the `inbound_number` make that difference? Since now we just throw things into redis here, do we need to move this test somewhere else?